functionsChallenges.js

```diff
@@ -0,0 +1,74 @@
+function maxOrMin(max, x, y) {
+    if (max && x > y) {
+        return x
+    } else if (max && y > x) {
+        return y
+    }
+}
+alert(maxOrMin(true,9,7))
+
+
+
+
+
+
+var arr = [1,4,6,8,9,7]
+function sillySum (array) {
+	var newArr =[]; 
+	for (var i = 0; i < array.length; i++) {
+		
+		 newArr.push(i*array[i]); 
```

Comment:
This looks awesome, but instead of pushing each "silly sum" into a new array, we want to keep a total sum of each array value times its index. Something like this:

```js
sillySum([1, 4, 6, 8, 9, 7]);
//=> 111
```